clove-unit.h

```diff
@@ -67,6 +67,7 @@ __CLOVE_EXTERN_C const char* __clove_get_exec_path();
 __CLOVE_EXTERN_C char* __clove_path_concat(const char separator, const char* path1, const char* path2);
 __CLOVE_EXTERN_C char* __clove_path_rel_to_abs_exec_path(const char* rel_path);
 __CLOVE_EXTERN_C bool __clove_path_is_relative(const char* path);
+__CLOVE_EXTERN_C char* __clove_path_to_os(const char* path);
```

Comment:
for now a `void __clove_path_to_os(char*)` could do the job, avoiding allocation